include/linker/common-rom.ld

```diff
@@ -181,6 +181,11 @@
 
 	Z_ITERABLE_SECTION_ROM(tracing_backend, 4)
 
+	SECTION_DATA_PROLOGUE(thread_info,,)
+	{
+		KEEP(*(".thread_info"));
```

Comment:
Can we have this be thread_dbg_info.  or at least add a comment here about what this is.  Since a section named thread_info makes me things its critical and not for debug.

Reply:
Good point, tried to find reasonable names with @tejlmand